timbermill-client/src/main/java/com/datorama/oss/timbermill/common/DbBulkRequest.java

```diff
@@ -0,0 +1,75 @@
+package com.datorama.oss.timbermill.common;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+import org.elasticsearch.action.bulk.BulkRequest;
+
+public class DbBulkRequest implements Serializable {
+	private String id;
+	private BulkRequest request;
+	private String createTime;
+	private Long insertTime;
+	private int timesFetched;
+
+
+	public DbBulkRequest(BulkRequest request) {
+		this.request = request;
+		String uniqueID = UUID.randomUUID().toString();
```

Comment:
I did it at the beginning but then I noticed that I couldn't know what is the ID on the client-side, and it makes inconsistency between the java object and the DB object - isn't it a door for bugs? also, it made it difficult for testing

Reply:
quite the opposite, it will ensure every insert will never fail, you want these kind of things to be handled by the database and not in code
https://www.tutorialspoint.com/sqlite/sqlite_using_autoincrement.htm

Reply:
ok changed it back, but if I want for example to log a message about a specific bulk - I used to specify the bulk id but now don't have it, what do we usually do in these cases?